### PR TITLE
Fix bungled normalization of plural/singular

### PR DESCRIPTION
**Pull request by ttung**
### Description
#85 was done incorrectly.  It's a Collection of SequencingReads.  Not a Collection of a singular SequencingRead.

#fml

### Test plan
upgraded and downgraded successfully.  repeated once more for good measure.

### CODE REVIEW

File: docs/backend/schema.ddl [hunk 1]
```diff
@@ -148,7 +148,7 @@ Table SequencingProtocol {
 
 // describes a single set of sequence data from the sequencer. (FASTQ)
 // these should be replaced by the `HostFilteredSequenceRead` as soon as it is available so we are not storing any host reads
-Table SequencingReadCollection {
+Table SequencingReadsCollection {
```
Comment (phoenixAja) [inline docs/backend/schema.ddl:151]:
```suggestion
Table SequencingReadsCollections {
```

File: src/py/aspen/database/models/sequences.py [hunk 2]
```diff
@@ -84,14 +90,14 @@ class SequencingProtocolType(enum.Enum):
...
-    __tablename__ = "sequencing_read_collections"
+class SequencingReadsCollection(Entity):
+    __tablename__ = "sequencing_reads_collections"
     __table_args__ = (UniqueConstraint("s3_bucket", "s3_key"),)
     __mapper_args__ = {"polymorphic_identity": EntityType.SEQUENCING_READS}
 
     entity_id = Column(Integer, ForeignKey(Entity.id), primary_key=True)
     sample_id = Column(Integer, ForeignKey(Sample.id), nullable=False)
-    sample = relationship(Sample, backref=backref("sequencing_reads"))
+    sample = relationship(Sample, backref=backref("sequencing_reads_collection"))
```
Comment (phoenixAja) [inline src/py/aspen/database/models/sequences.py:100]: should this be `sequencing_reads_collections`?

Comment (ttung) [reply src/py/aspen/database/models/sequences.py:100]: Miraculously, this is correct.  For any given sample, there can be at most one `sequencing_reads_collection`.

Comment (phoenixAja) [reply src/py/aspen/database/models/sequences.py:100]: ok interesting lol

**Review by phoenixAja — APPROVED** [review]

**Pull request merged**